index.js

```diff
@@ -2,19 +2,516 @@ const readline = require('readline');
+	} else if (roomInv.length > 0 && collectedItem.item === userAnswer) {
+		roomInv.splice(roomInv.indexOf(collectedItem));
+		playerInventory.push(collectedItem);
+		console.log(`You collected a ${collectedItem.item}!`);
+	} else {
+		console.log("I don't know what you want...");
+	}
+};
+
+/*-------------global drop function that will remove item from players inventory and drop it into the rooms inventory-------------------*/
+let drop = function () {
+	for (let items of roomInv) {
+		collectedItem = items;
+	}
+
+	for (let items of playerInventory) {
+		droppableItems = items;
+	}
+	if (immovableItemsTable.includes(userAnswer)){
+    console.log(`No! i will not drop my precious ${userAnswer}`)
+  }else if(playerInventory.length > 0) {
+		droppableItems.item.includes(userAnswer);
+		playerInventory.splice(playerInventory.indexOf(droppableItems));
+		roomInv.push(droppableItems);
+		console.log(`You dropped a ${droppableItems.item}`);
+	} else {
+		console.log('I cant...');
+	}
+};
+
+//--------Global look around function to be called on in any room --------//
+let lookAround = function () {
+	if (roomInv.length !== 0) {
+		for (let roomItems of roomInv) {
+			collectedItem = roomItems.item;
+		}
+		console.log('\n' + roomDescription);
+		for (let roomItems of roomInv) {
+			console.log('you see a ' + roomItems.item);
+		}
+	} else if (roomInv.length === 0) {
+		console.log(roomDescription);
+		console.log("There's nothing here..");
+	}
+};
+
+//-------global check inventory function that will take the items in the player inventory array and display there item name.
+let checkInventory = function () {
+	if (playerInventory.length > 0) {
+		for (let items of playerInventory) {
+			droppableItems = items;
+			console.log(droppableItems.item + ': ' + droppableItems.description);
+		}
+	} else if (playerInventory.length <= 0) {
+		console.log('Empty');
+	}
+};
+
+//-----making a list of room descriptions to call on inside global look around functions
+roomDescription = undefined;
+let lakeRoomDescription =
+	'You are in a clearing with a small lake.. the water looks questionable, but you are so terribly thirsty...\n';
+let riverRoomDescription = 'There is massive river, obstructing all movement except to the north\n';
+let meadowRoomDescription =
+	'A foreboding mountain lies to the East and West. A path cuts through the woods to the north, and there is the sound of a river to the south\n';
+let puzzleRoomDescription =
+	'You come to a forrest. It is shady aside from some slivers of light breaking through the leaves.';
+let deadEndRoomDescription = 'You now stand at the edge of a giant cliff.';
+let wellRoomDescription = 'You are in a field with a well.';
+let lockedRoomDescription =
+	'You now come to a gate, guarded by a troll. The troll will not let you pass, unless he is paid off.';
+let finalRoomDescription =
+	'You are in a cave. There is a portal to the south, which appears to lead home, but not much else.';
```

Comment:
There were a few times where I had to refer back to the code itself to remember what my available options were for alternate paths. Adding a bit more to some of your room descriptions would make navigating a bit easier